src/components/Session/index.js

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import Button from "react-bootstrap/Button";
+import Card from "react-bootstrap/Card";
+import { selectUser } from "../../store/user/selectors";
+import { utcToZonedTime } from "date-fns-tz";
+import { useSelector, useDispatch } from "react-redux";
+import { bookSession, unBookSession } from "../../store/session/actions";
+import { NavLink } from "react-router-dom";
+
+
+export default function Session({ session, teacher, btn, hideSubject,bg }) {
```

Comment:
space at the end there after the `,`. I think you're prettier might not be working